src/main/java/org/desarrolladorslp/technovation/services/impl/DeliverableServiceImpl.java

```diff
@@ -181,6 +202,19 @@ public void assignDeliverableToSession(UUID deliverableId, UUID sessionId, Relat
             throw new SessionDoesNotBelongToBatch(sessionId + "does not belong to the same batch as deliverable");
     }
 
+    @Override
+    public List<Resource> getResourcesByDeliverable(UUID deliverableId) {
+        return deliverableRepository.getResourcesByDeliverable(deliverableId);
+    }
+
+    @Override
+    @Transactional
+    public void deleteResourceFromDeliverable(UUID deliverableId, UUID resourceId) {
```

Comment:
We need to validate that this operation is performed by the owner of the deliverable